runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpPanelLabel.java

```diff
@@ -45,8 +42,25 @@
 	;
 
 	private final String key;
+	private final Function<XpSnapshotSingle, String> keyFunc;
```

Comment:
Added `@Nullable` for clarity
```suggestion
	@Nullable
	private final Function<XpSnapshotSingle, String> keyFunc;
```